ui/src/pages/publications/[id]/index.tsx

```diff
@@ -52,7 +52,7 @@ export const getServerSideProps: Types.GetServerSideProps = async (context) => {
 
     try {
         const response = await api.get(`${Config.endpoints.publications}/${requestedId}/bookmark`, token);
-        bookmark = response.data;
+        bookmark = response.data ? true : false;
```

Comment:
if this is meant to be a boolean, then it should be called **isBookmarked** or something similar

Reply:
Renamed to isBookmarked